README.md

```diff
@@ -37,48 +34,44 @@ kro is Kubernetes native and integrates seamlessly with existing tools to preser
-    You can just as easily create a more complex *WebAppWithDB* resource group by combining the existing *WebApp* resource group with a *Table* custom resource to provision a cloud managed database instance for your web app to use.
+    For example, you can define a *WebApp* ResourceGraphDefinition that defines a *WebAppCRD* CRD that is composed of a *Deployment*, pre-configured to deploy your web server backend, and a *Service* configured to run on a specific port.
+    You can just as easily create a more complex *WebAppWithDB* ResourceGraphDefinition by combining the existing *WebApp* ResourceGraphDefinition with a *Table* custom resource to provision a cloud managed database instance for your web app to use.
 
-    Once you have defined a ResourceGraphDefinition, you can apply it to a Kubernetes cluster where the kro controller is running.
-    kro will take care of the heavy lifting of creating CRDs and deploying dedicated controllers in order to manage instances of your new custom resource group.
+    Once you have defined a ResourceGraphDefinition, you can apply it to a Kubernetes cluster where the kro controller is running. kro will take care of the heavy lifting of creating CRDs and deploying dedicated controllers in order to manage instances of your new custom resource group.
 
-    You are now ready to create instances of your new custom resource group, and kro will respond by dynamically creating, configuring, and managing the underlying Kubernetes resources for you.
+    To create an instance of your custom resource groupings, you create an instance of the CRD that your ResourceGraphDefinition specifies. In the WebApp ResourceGraphDefinition example, this would be an instance of the *WebAppCRD* CRD. kro will respond by dynamically creating, configuring, and managing the underlying Kubernetes resources for you. 
 
 4. **Why did you build this project?**
 
-    We want to help streamline and simplify building with Kubernetes.
-    Building with Kubernetes means dealing with resources that need to operate and work together, and orchestrating this can get complex and difficult at scale.
-    With this project, we're taking a first step in reducing the complexity of resource dependency management and customization, paving the way for a simple and scalable way to create complex custom resources for Kubernetes.
+    We want to help streamline and simplify building with Kubernetes. Building with Kubernetes means dealing with resources that need to operate and work together, and orchestrating this can get complex and difficult at scale.
+   With this project, we're taking a step in reducing the complexity of resource dependency management and customization, paving the way for a simple and scalable way to create complex custom resources for Kubernetes.
 
-5. **Do I need to have an AWS account to use this?**
+6. **Do I need to have an AWS / Google Cloud / Azure account to use this?**
```

Comment:
```suggestion
5. **Do I need to have an AWS / Google Cloud / Azure account to use this?**
```
same for number 7 and 8

Reply:
I just cut a quick PR on the repo README to drop the "Do I need an account" and "Will this be built into the service" questions altogether. I'm ok either way.